### ISSUE TITLE
Please consider changing delete icon / placement + adding a confirmation for deleting a widget list item

### ISSUE THREAD

**Issue by squidgemonster** [decaporg/decap-cms#7135]
I haven't found anything relating to this searching through issues - but to give a bit of context:

_A new client got confused while adding items to a widget list. They clicked 'x' on one, thinking it was going to close the accordion for that item - which isn't too much of a stretch - didn't see the UI change - and consequently lost that entry._

Please consider:

- Introducing a delete confirmation dialog (even just a simple confirm() dialog) on clicking the delete icon, to make it clear this action is taking place / avoid accidental removal of items
- Changing the 'x' icon to a trash icon instead, to make it clearer that it's a delete action that will be triggered
- Alternatively - instead of a 'x' icon - put a 'Delete' button within the open accordion panel, in the lower right, removing the need for an icon on the closed item, and also moving the delete action further away from the 'Add' button at the top of the list widget, which it also top right.

Hopefully there is some logic in there. I would be happy to offer some UX input into the project, if it's needed. Sadly my coding skills don't stretch to React, to contribute to the code directly.

**Comment by martinjagodic** [comment]
@squidgemonster I agree with replacing the icon from X to the trash bin, but I am skeptical about an alert prompt because it could slow down a bulk operation.

**Comment by squidgemonster** [comment]
@martinjagodic I think the icon would make a big difference, in the first instance.

Bulk operations could be a toggle, in that case... default behaviour is a prompt, if you switch to batch delete, then you can click and delete instantly?
